draft-ietf-scitt-architecture.md

```diff
@@ -297,7 +297,7 @@ Transparency is implemented by providing a consistent, append-only, cryptographi
 A SCITT instance is referred to as a Transparency Service.
 Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
 A Receipt is an offline, universally-verifiable proof that an entry is recorded in the Append-only Log.
-Receipts do not expire, but it is possible to append new entries (more recent Signed Statements) that subsume older entries (less recent Signed Statements).
+Requesting a refreshed receipt can result in a new receipt over the same signed statement, with an updated signature, signing algorithm, and new key material expiration.
```

Comment:
Suggestion, based on merged text consistency:
```suggestion
Verifiers can request refreshed receipts resulting in a new receipt over the previously signed statement, with an updated signature, signing algorithm, and new key material expiration.
```

While this wording is a bit more consistent, it does raise the question:
Is the new, refreshed receipt persisted in the append-only log? Is the premise that anyone can request a refreshed receipt?

- **Jan 1, 2025**: Registration of "foo" is done, and a receipt is issued
- **Jan 2, 2025**: a request for the receipt is made, and it's still valid, so it's not "refreshed", rather the existing receipt is returned.
- **Feb 1, 2025**: the key material of the original receipt expired, so the verifier requests a refreshed receipt with an updated signature expiration.
- **Feb 2, 2025**:, another verifier requests the receipt. Do they get the originally signed and expired signature receipt, or the refreshed receipt generated the day prior? 

Or: 

1. Is the behavior that any request of an existing receipt is considered a READ scenario, and doesn't necessarily require RBAC? If it's expired, it's automatically refreshed, but not durable? 
1. Can verifiers request existing expired receipts, which stay expired until someone with permissions can refresh the signature of the receipt.
1. If requesting a refreshed receipt is durable, this implies a set of permissions (RBAC) to write back.

Conceptually, I get the desired intent. In practice, this seems to require more discussion.

Reply:
We abstained from defining Verifier, right?

Reply:
Yeah, I was looking for a better way to make this sentence fit within the paragraph. They read as isolated bullets, rather than a paragraph.

Reply:
resolving this thread to propose an alternative